Source/KakapoServer.swift

```diff
@@ -8,21 +8,42 @@
 
 import Foundation
 
+/**
+ A RouteHandler used when registering different HTTP methods. This can return any Serializable object, though you would normally want to use the built-in Response object
+ */
```

Comment:
Not sure what you mean... Also mention that returning a `Response` you can provide status code and headers otherwise will use 200 by default